app/src/main/java/org/mozilla/fenix/ext/Dialog.kt

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ext
+
+import android.app.Activity
+import android.app.Dialog
+import android.view.WindowManager
+
+/**
+ * Checks if activity's window has a FLAG_SECURE set and sets it to dialog
+ */
+fun Dialog.secure(activity: Activity?) {
```

Comment:
This would be cool to have something like this in [`feature-privatemode`](https://github.com/mozilla-mobile/android-components/tree/master/components/feature/privatemode). 🙂

Reply:
👍 added it to the upstreaming board! 

https://github.com/orgs/mozilla-mobile/projects/38